client/state/reader/start/reducer.js

```diff
@@ -85,8 +118,34 @@ export function recommendationsInteractedWith( state = [], action ) {
 	return state;
 }
 
+/**
+ * Returns the state of the user's Reader. New users are shown the cold start
+ * recommendations, while regular / graduated users are shown the normal recs.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export function isNewReader( state = false, action ) {
+	switch ( action.type ) {
+		case READER_START_GRADUATE_REQUEST_SUCCESS:
+			return false;
+
+		case READER_START_GRADUATE_REQUEST_FAILURE:
+			return true;
+
+		case SERIALIZE:
+		case DESERIALIZE:
+			return false;
```

Comment:
probably want the default to be `null`, not `false`, as it'll let you fall through.